tests/test_core.py

```diff
@@ -375,7 +376,7 @@ def test_region_processing_skip_aggregation(model_name, region_names):
             [["m_a", "s_a", "World", "Variable B", "EJ/yr", 4, 6]],
             [
                 "Difference between original and aggregated data:",
-                "m_a   s_a      World  Variable B EJ/yr 2005         4           3",
+                "m_a   s_a      World  Variable B EJ/yr 2010         6",
```

Comment:
Why did the value change?

Reply:
It's because I now sort the differences by the relative difference and the most severe one is for 2010 and no longer for 2005. This is what the full warning message looks like:
```console
2023-05-26 10:39:50 WARNING  Difference between original and aggregated data:
                                             original  ...  difference (%)
model scenario region variable   unit  year            ...                
m_a   s_a      World  Variable B EJ/yr 2010         6  ...       33.333333
                                       2005         4  ...       25.000000
```
I just don't test for the entire message